web/src/components/organisms/Workspace/Settings/hooks.ts

```diff
@@ -0,0 +1,61 @@
+import { useCallback } from "react";
+import { useNavigate } from "react-router-dom";
+
+import Notification from "@reearth-cms/components/atoms/Notification";
+import {
+  useUpdateWorkspaceMutation,
+  useDeleteWorkspaceMutation,
+} from "@reearth-cms/gql/graphql-client-api";
+import { useT } from "@reearth-cms/i18n";
+import { useWorkspace } from "@reearth-cms/state";
+
+export default () => {
+  const navigate = useNavigate();
+  const [currentWorkspace, setCurrentWorkspace] = useWorkspace();
+  const t = useT();
+
+  const workspaceId = currentWorkspace?.id;
+  const workspaceName = currentWorkspace?.name;
+
+  const [updateWorkspaceMutation] = useUpdateWorkspaceMutation();
+  const [deleteWorkspaceMutation] = useDeleteWorkspaceMutation({
+    refetchQueries: ["GetMe"],
+  });
+
+  const handleWorkspaceUpdate = useCallback(
+    async (name?: string) => {
+      if (!workspaceId || !name) return;
+      const workspace = await updateWorkspaceMutation({
+        variables: {
+          workspaceId,
+          name,
+        },
+      });
+      if (workspace.errors || !workspace.data?.updateWorkspace) {
+        Notification.error({ message: t("Failed to update workspace.") });
+        return;
+      }
+      setCurrentWorkspace({ ...workspace.data.updateWorkspace.workspace });
+      Notification.success({ message: t("Successfully updated workspace!") });
```

Comment:
I think I have let through this quite often, so I apologize for suddenly asking for a change, but I feel like if/else is more clear and understandable then using the return. Can you update where you wrote this in this PR and then while doing other tasks if you come across this code can you slowly update?

Reply:
@KaWaite , In this case in particular I disagree, I feel the `return` style of coding when something is wrong is better than `if/else`
Should we take advice and ask Inouse-san?
If not let me know, and I'll change it.

Reply:
@issmail-basel Honestly both get the job done. Just in this case I think we usually do if/else, and you are actually doing if/else in the handleWorkspaceDelete function right below this function. 
Please change here and we can discuss more later in the week if you want to.

Reply:
@KaWaite , sure, I changed it.